lib/galaxy/dependencies/conditional-requirements.txt

```
@@ -15,6 +15,7 @@ galaxycloudrunner
 pkce
 total-perspective-vortex>=2.3.2,<3
 openai
+redis==5.3.0
```

```suggestion
redis>=5.3.0
```
should be fine I think. I just didn't want to test that out in production :)